lunatrace/bsl/backend/src/types/sqs.ts

```diff
@@ -25,7 +25,25 @@ export interface QueueHandlerConfig {
 
 export type BuildSourceType = 'pr' | 'gui' | 'cli';
 
-export interface GenerateSnapshotForRepositoryRecord {
+export type LunaTraceSqsEventType = 'repository-snapshot' | 'process-webhook';
```

Comment:
`type LunaTraceSqsEventType = LunaTraceSqsEvent['type']` If you want to be fancy.